src/worker/tailwind.config.js

```diff
@@ -9,15 +9,17 @@ module.exports = {
   theme: {
     extend: {
       fontSmoothing: {
-        antialiased: true,
+        antialiased: true
+      },
+      screens: {
+        md: '745px',
```

Comment:
empircally determined - sensitive to mx-3 margin and width of monospace `0`.
no biggie if it's a bit off.